docker-compose/glances.conf

```diff
@@ -424,6 +424,8 @@ max_name_size=20
 all=False
 # Define Podman sock
 #podman_sock=unix:///run/user/1000/podman/podman.sock
+# Define Docker sock
+docker_sock=unix:///var/run/docker.sock
```

Comment:
It will be better to let default value of docker_sock to empty (not defined) in order to use (by default) the standard docker.from_env() builder. 
Glances is multi-platform and i am pretty sure that the default value of docker_sock on Windows is not unix:///var/run/docker.sock.

Reply:
+1 for this